rllib/algorithms/dqn/dqn.py

```diff
@@ -497,6 +502,21 @@ def get_default_rl_module_spec(self) -> RLModuleSpec:
                 "Use `config.framework('torch')` instead."
             )
 
+    # TODO (sven, simon): We cannot use our `override` decorator here, as the
```

Comment:
I think this does work. E.g.:
```
from ray.rllib.algorithms.algorithm_config import AlgorithmConfig
from ray.rllib.utils.annotations import override


class MyConfig(AlgorithmConfig):
    @property
    @override(AlgorithmConfig)
    def rl_module_spec(self):
        return 1


A = MyConfig()
print(A.rl_module_spec)
```

Reply:
Alright, I changed the order of context.